src/particle_header.F90

```diff
@@ -232,7 +232,7 @@ subroutine create_secondary(this, uvw, type, run_CE)
     this % n_secondary = n
     this % secondary_bank(this % n_secondary) % E = this % E
     if (.not. run_CE) then
-      this % secondary_bank(this % n_secondary) % E = real(this % g, 8)
+      this % secondary_bank(this % n_secondary) % E = real(this % g,8)
```

Comment:
Comma should [remain](http://openmc.readthedocs.io/en/latest/devguide/styleguide.html#whitespace-in-expressions).